Deploying_vLLM/vllm_autocalc.py

```diff
@@ -332,7 +346,20 @@ def main():
     # Overwrite params then perform autocalc
     fd = overwrite_params(fd)
     try:
-        output_dict = vllm_auto_calc(fd)
+        if fd['MAX_MODEL_LEN_CONFIG'] != 0:
+            print('\nRecipe Calc - 1st Iteration *******************')
+            fd['MAX_MODEL_LEN'] = fd['MAX_MODEL_LEN_CONFIG']
+            fd['MAX_NUM_SEQS_CONFIG'] = 0
```

Comment:
Remove MAX_NUM_SEQS_CONFIG, we will use user provided vLLM param MAX_NUM_SEQS for override